sdk/attestation/attestation/src/models/attestationToken.ts

```diff
@@ -4,16 +4,28 @@
  *
  */
 
+//import {JsonWebKey} from "../generated/models";
```

Comment:
a lot of commented code here, please remove it if not needed.

Reply:
It will be needed eventually - I just didn't want to spend the time to figure out how to construct a JSON Web Key from the JSON encoded JWK. I'll eventually fill this in.